ValheimPlus/Configurations/Sections/KilnConfiguration.cs

```diff
@@ -2,7 +2,7 @@
 {
     public class KilnConfiguration : ServerSyncConfig<KilnConfiguration>
     {
-        public float productionSpeed { get; internal set; } = 10;
+        public float productionSpeed { get; internal set; } = 16;
```

Comment:
Should be 15 as per the default game settings

Reply:
Thanks for checking, you're right!